components/bullet/theme.css

```diff
@@ -9,6 +9,14 @@
   border-radius: 50%;
   display: inline-block;
   vertical-align: middle;
+
+  & + h1,
+  & + h2,
+  & + h3,
+  & + h4,
+  & + p {
+    margin-left: 6px;
+  }
```

Comment:
Is it me or does this feel iffy?

Reply:
The spacing between Bullets and inline text was defined in the styleguide so I wanted to include it in the component. As I think more about it now, it might not make that much sense anymore as we force people to combine it in a certain way.

Reply:
Exactly! Components should be standalone, not sure how we can fix this properly though.

Reply:
<Heading1WithBullet />

Reply:
I reverted the commits where I change the structure of the example and the weird spacing CSS so this PR only focuses on the global styles it added.